From 833c8c20a25235dfd08a2ecc80a6bef95c9a73ed Mon Sep 17 00:00:00 2001
From: Xinyu Chen <Xinyu.Chen@freescale.com>
Date: Fri, 24 Oct 2008 14:52:10 +0800
Subject: [PATCH] ENGR00096532 MX35 Change LCD pixel clock rate to 25MHz

Change LCD pixel clock back to 25MHz
Remove such workaround

Signed-off-by: Xinyu Chen <Xinyu.Chen@freescale.com>
---
 drivers/video/mxc/mxcfb_claa_wvga.c |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/drivers/video/mxc/mxcfb_claa_wvga.c b/drivers/video/mxc/mxcfb_claa_wvga.c
index 2c1435e..a9e18c4 100644
--- a/drivers/video/mxc/mxcfb_claa_wvga.c
+++ b/drivers/video/mxc/mxcfb_claa_wvga.c
@@ -48,7 +48,7 @@ static int lcd_on;
 static struct fb_videomode video_modes[] = {
 	{
 	 /* 800x480 @ 55 Hz , pixel clk @ 25MHz */
-	 "CLAA-WVGA", 55, 800, 480, 50000, 80, 0, 10, 0, 20, 10,
+	 "CLAA-WVGA", 55, 800, 480, 40000, 80, 0, 10, 0, 20, 10,
 	 FB_SYNC_OE_ACT_HIGH,
 	 FB_VMODE_NONINTERLACED,
 	 0,},
-- 
1.5.4.4

